.gitignore

```diff
@@ -17,7 +17,10 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+.eslintcache
+.vscode
```

Comment:
So, I am not sure if this is an attempt to stop future changes, or stop the changes you have added here in the branch, but since `.vscode/extensions.json` and `.vscode/settings.json` have been checked in already, I think any changes made in the future will still be permitted. Ie, the gitignore has no effect on them.